### DISCUSSION TITLE
各位记得避雷一下这位绳匠

### DISCUSSION THREAD

**Discussion by Prance36** [share121/inter-knot#4916] [其他]
就是这位@BJ-cmyk，我让这哥们去找我的货，结果他给找错箱子了，气死我了，白瞎我的定金了。大家发委托的时候一定要记得避雷，这种眼神不好使的就不要随便请了

**Comment by BJ-cmyk** [comment]
你敢质疑我家AI的实力？！😡你明明买的就是邦布斐济杯插件！😡...欸～有没有可能，箱子被匪帮掉包了！🤓
![1723465325409](https://github.com/user-attachments/assets/3cba060d-6c01-4522-9b73-5eeafc14885c)

**Reply by bigetree** [reply]
买的...什么？

**Comment by March-777th** [comment]
不是戈门<img src="https://bbs-static.miyoushe.com/static/2024/07/31/73346f2b98d45b62da9139d2774fa856_8641114165088315754.png" alt="三月七 告退" width="75" >你买的什么？

**Comment by Prance36** [comment]
各位，我买的插件不是斐济杯呀！！！
就是普通的工程用邦布插件

**Comment by Minapocalis** [comment]
什么杯？干什么用的

**Comment by zzz20100624** [comment]
楼主，你的斐济杯插件有链接吗🤓👆

**Reply by Prance36** [reply]
没有，我买的根本就不是斐济杯，我买的就是普通的工程用插件
